### PR TITLE
added snackbar components

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please Review**

snackbar

### CODE REVIEW

File: Components/Snackbar/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Ansh Tripathi">
+    <meta name="description" content="Snackbar components">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (vikrantkode) [inline Components/Snackbar/snackbar.html:12]: import in index.js and then use it

**Review by vikrantkode — APPROVED** [review]
good to go

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Review by JatuFaique — COMMENTED** [review]
clean code practices followed
    good folder structure
    good to merge

**Pull request merged**